frontstage_api/controllers/case_controller.py

```diff
@@ -0,0 +1,29 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from frontstage_api import app
+from frontstage_api.common.request_handler import request_handler
+from frontstage_api.decorators.api_error_handler import api_error_handler
+from frontstage_api.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@api_error_handler
+def get_case_by_party_id(party_id):
+    logger.debug('Retrieving case', party_id=party_id)
+    url = app.config['RM_CASE_GET_BY_PARTY'].format(party_id)
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code == 204:
+        logger.warning('No case found for party id', party_id=party_id)
+        raise ApiError('FA200')
```

Comment:
We should avoid using arbitrary codes

Reply:
This was a pattern that was discussed with @pricem14pc and @rjweeks. We should probably review it here to make sure it's what we want before adding more to the API though